pyproject.toml

```diff
@@ -115,3 +115,21 @@ testpaths = [
     "src/py/flwr",
     "src/py/flwr_tool",
 ]
+
+[tool.mypy]
+ignore_missing_imports = true
+strict = true
+
+[[tool.mypy.overrides]]
+module = "flwr_example.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "flwr_experimental.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "torch.*"
+follow_imports = skip
```

Comment:
CI is failing and I think its because of this line. Did you try running `dev/test.sh` locally?

Reply:
```
follow_imports = "skip"
```
might work but I am not sure.

Reply:
Ok, I'll try `follow_imports = "skip"`

No, I didn't run `dev/test.sh` locally

Reply:
My recommendation would be try running the mypy part in "dev/test.sh" or generally all tests. This makes it easier to detect errors before it hits the CI.

Reply:
Thanks for the recommendation @tanertopal 

Here's what I did:

-  tried running `dev/test.sh` It basically stops after printing this: "17 files would be reformatted, 51 files would be left unchanged." It broke somewhere in between or else the message should have been "All Python checks passed"

- tried running `python -m mypy src/py ` as you also suggested that running just mypy part would also do. Again, I had lots of errors revolving around "Library stubs not installed for "google.protobuf.*******"

Now, I have few questions to ask:

To make sure that I have the right environment to run the test, how can I create one? What I'm looking for is something like requirements.txt, which I directly pip install to get all the packages.

Also, does the operating system plays any part here? 

Also, if you could share any links or references to create the environment for flower that'd be highly appreciated. 

Thanks again )

Reply:
@architjen so to create a correct environment you can run sequentially the following scripts:

* venv-create.sh
* bootstrap.sh

The first one you may skip if you take care of a venv yourself. The second script will install everything including poetry into your environment.

Afterwards you can use mostly all other scripts. Does this help?

Reply:
Hi @tanertopal, nope it doesn't seem to help. I even tried running `poetry install` so that it creates a virtual env with all dependencies. 

I tried testing on different branches as well.

I don't know what else can be done: Maybe deleting my local repo and re-git cloning?

Reply:
@architjen I checked your PR and it looks good so I am going to merge it so #762 is resolved. Thank you for your contribution. We'll create a contributor guide to explain the dev setup in a bit more details in the foreseeable future.  If you are still interested to setup the repo so you can contribute further let us continue the discussion on Slack?

Reply:
@tanertopal thanks for the merge. 

Yeah, it'll be nice to have a guide on dev setup, maybe I can help there as well?

Sure, we can take it to Slack, I'm looking forward to more contributions, having a setup would be requisite.